### ISSUE TITLE
Can I use ananse to compare TFs networks between disease and normal condition such as aging and young ,or fibrosis disease and healthy control?

### ISSUE THREAD

**Issue by socialtree-yt** [vanheeringen-lab/ANANSE#209]
Can I use ananse to compare TFs networks between disease and normal condition such as aging and young ,or fibrosis disease and healthy control rather than differentiation?

**Comment by siebrenf** [comment]
Hi socialtree,

> Can I use ananse to compare TFs networks between disease and normal condition such as aging and young ,or fibrosis disease and healthy control

ANANSE network outputs a network per condition and ANANSE influence outputs a differential network. However, these networks are very tricky to analyse. The top influential TFs output by ANANSE influence are much more robust.

> rather than differentiation

I'm not sure I understand this part. In the final step, ANANSE influence, we require a Differential Gene Expression analysis as input. This data is combined with the networks from the previous step, ANANSE network, to find the top influential TFs, which are something more than "just" differentiation.

> if I use narrowPeak file in ananse binding -r parameter, will it calculates binding results in enhancer region automatically or just calculate merged regions among narrowPeaks?

With the `-r` parameter, you tell ANANSE in which regions to look. The enhancer activity within those regions is automatically calculated from the enhancer data, which you give with the `-A`, `-H`, or `-C` parameters, depending on your data type (ATAC-, H3k27Ac ChIP-, or CAGE-seq respectively)

**Comment by socialtree-yt** [comment]
_"With the -r parameter, you tell ANANSE in which regions to look"_
I also want to know the BED format for this parameter. Can I use BED6 format or must with BED12 format? And if I have predefined an enhancer region bed file containing typical enhancer and super enhancer. No matter how ananse identify the genomic location of these region, ananse would calculate the activity in my primary region but not split my super enhancer into multiple regions or discard these regions. Is it right?

**Comment by socialtree-yt** [comment]
_If this does not work for your analysis, you could supply a pfmscorefile to ANANSE binding. See ananse binding --help and gimme scan --help for more info._
I think this method helps to predict motif bindings according to different database but not helps to solving the scaled same width?

**Comment by socialtree-yt** [comment]
If this does not work for your analysis, you could supply a pfmscorefile to ANANSE binding. See ananse binding --help and gimme scan --help for more info.
I think this method helps to predict motif bindings according to different database but not helps to solving the scaled same width if I need regions with primary and different width?

**Comment by socialtree-yt** [comment]
_If this does not work for your analysis, you could supply a pfmscorefile to ANANSE binding. See ananse binding --help and gimme scan --help for more info._
I think this method helps to predict motif bindings according to different database but not helps to solving the scaled same width if I need regions with primary and different width?

**Comment by socialtree-yt** [comment]
And I see "H3K27ac signal, for instance, would not work well, as peaks from a H3K27ac ChIP-seq experiment are too broad to provide a precise region for the motif analysis. You can also provide one or more narrowPeak files, for instance from [MACS2](https://github.com/taoliu/MACS)." I want to know whether it's precise to use narrowPeak of CHIP seq data as regions to do ananse binding or ATAC narrow peak works better? Can I use both ATAC and CHIP seq narrowPeak as regions? Does it outperforms than only with ATAC narrow peak?
Thank you for your help!

**Comment by socialtree-yt** [comment]
And for example, if I choose a wide range A in "ananse network -r parameter", does all regions in "binding.h5" file of ananse binding results will be selected if they are contained in range A? Or I must select regions as same as regions in "binding.h5" file?

**Comment by siebrenf** [comment]
> I want to know whether it's precise to use narrowPeak of CHIP seq data as regions to do ananse binding or ATAC narrow peak works better? Can I use both ATAC and CHIP seq narrowPeak as regions? Does it outperforms than only with ATAC narrow peak?

Yes, yes and yes! NarrowPeak files to indicate regions are good for both ChIP-seq and ATAC-seq, and using both H3K27Ac ChIP-seq BAMs with ATAC-seq BAMs is better than either one alone.

> if I choose a wide range A in "ananse network -r parameter", does all regions in "binding.h5" file of ananse binding results will be selected if they are contained in range A? Or I must select regions as same as regions in "binding.h5" file?

The latter. `ananse network -r` is used to filter your regions. So you can only select regions from the "binding.h5" file.

> I think this method [...] not helps to solving the scaled same width if I need regions with primary and different width?

I believe ANANSE gives the best results if all input regions have the same width. 

You could to this by taking the `start` + `peak` values from all your narrowPeak [format](https://www.biostars.org/p/102710/) files, and extending each region around this coordinate. Which width you use if up to you (for reference: our default is 200 bp).

My advise would be to try this option first, and then see what the effect of using variable region widths is (I expect that scores become biased towards wider regions).

I hope this is more clear :)

**Comment by socialtree-yt** [comment]
_You could to this by taking the start + peak values from all your narrowPeak [format](https://www.biostars.org/p/102710/) files, and extending each region around this coordinate. Which width you use if up to you_
OK. Thank you! It's very clear but I also have a question that whether ANANSE will scale lengths of regions automatically to 200bp if I use narrowpeak as input regions because narrowpeak usually has different region lengths. And if I extend my region as you suggest, will the regions be scaled to 200bp?
